fuel.py

```diff
@@ -0,0 +1,23 @@
+while True:
+    try:
+        # get x and y
+        x, y = input("Fraction: ").split("/")
+        x = int(x)
+        y = int(y)
+
+        percentage = round((x / y) * 100)
+        if percentage <= 1:
+            print("E")
+        elif percentage >= 99:
+            print("F")
+        else:
+            print(f"{percentage}%")
+
+        if x > y or y == 0:
```

Comment:
It's better to check those condition earlier, before you run x/y so that you have more control over your code.

The y==0 check will be never true here, because in case of y=0, the ZeroDivisionError will be thrown in line 8.